### DISCUSSION TITLE
Is it possible to use two devices (mbus & Wmbus)?

### DISCUSSION THREAD

**Discussion by Psicrow925** [wmbusmeters/wmbusmeters#1206] [Q&A]
Hello
I would like to read two meters.

One via mbus and one via wmbus.

But if I specify two devices on the config page, after saving and restarting, one seems to be ignored or overwritten and I get no telegrams from both?

I use Home Assistant.

**Comment by weetmuts** [comment]
Can you post your configuration here? It should be possible to read both mbus and wmbus. 

From the readme, you have give the mbus a name if you have multiple buses, eg from a config:
device=MAIN=/dev/ttyUSB0:mbus:2400
device=RADIOMAIN=/dev/ttyUSB1:im871a:c2

And then the mbus meter has to have its pollinterval set.
pollinterval=60s

And the mbus driver must look like:
driver=themeterdriver:MAIN:mbus

Otherwise wmbusmeters does not know which device to send the poll request to.

**Comment by Psicrow925** [comment]
Thank you for your reply. 

I will try it out this evening.

**Comment by Psicrow925** [comment]
Hello

Unfortunately I have the same result as yesterday. a device is deleted after saving and reloading.


![Screenshot 2024-03-08 165420](https://github.com/wmbusmeters/wmbusmeters/assets/63744267/a9767b94-f2a6-4ac0-912f-5ba364ba2d17)


After saving and reloading, it will look like this.


![Screenshot 2024-03-08 165529](https://github.com/wmbusmeters/wmbusmeters/assets/63744267/effd86f3-d9fe-4108-b9f4-e50c79850604)



if i configure mbus individually, everything works and i get values.
the same applies if i configure wmbus individually, then i receive the encrypted telegrams.

but both devices cannot be configured

**Comment by Psicrow925** [comment]
I have found the answer here
[2 Devices Configuration Home Assistant](https://github.com/wmbusmeters/wmbusmeters/issues/1076)

![Screenshot 2024-03-08 201948](https://github.com/wmbusmeters/wmbusmeters/assets/63744267/c2cad52b-b936-40b5-9df1-3345e24a0ea3)

This works for me
